src/Jaeger.Core/Reporters/RemoteReporter.cs

```diff
@@ -160,9 +165,9 @@ public AppendCommand(RemoteReporter reporter, Span span)
                 _span = span;
             }
 
-            public void Execute()
+            public Task ExecuteAsync()
             {
-                _reporter._sender.Append(_span);
+                return _reporter._sender.AppendAsync(_span, CancellationToken.None);
```

Comment:
I don't like to have `CancellationToken.None` in library code since it somewhat breaks the token flow. Usually it's better to just use the method variant without `CancellationToken`. Since this method is defined in `ISender` and without alternative, I proposed to change `ISender` to use `CancellationToken cancellationToken = default` above. The implementing classes don't have to be changed than and it behaves exactly as with `CancellationToken.None`  being used here.

Reply:
See my previous comment on why I disagree. Instead we should decide if it makes sense to provide an actual timeout-token here.